Multiplexed data (#92)
nf-core/test-datasets: README.md

| File	                | Description	                                                                                              |
|-----------------------|-----------------------------------------------------------------------------------------------------------|
| `samplesheet.csv`     | Sample information file                                                                                   |

> | `fast5/`              | Subset of fast5 files from direct cDNA Nanopore reads for HepG2 (Liver Cancer) and K562 (Leukemia) cell lines                  |

Reviewer: Add a listing of both directories in `fast5`  i.e. `fast5/barcoded/` and `fast5/nonbarcoded/`

Reviewer: Ive changed this now but Im still unsure which cell lines the `fast5` data from the `barcoded` and `non-barcoded` data come from. Are they from the same samples? Maybe add a more relevant description for each in the `Files` section. Also, could rename the sample in `samplesheet_nonbarcoded.csv` to something a bit more accurate. Ive put in `K562_RUN1_REP3` for now as a placeholder.

[Read more about SG-NEx](https://github.com/GoekeLab/sg-nex-data)

> There are 2 test configs - 1 for barcoded data and one for non-barcoded data

Reviewer: ```suggestion
There are 2 test-datasets- 1 each for barcoded and non-barcoded data, respectively.
```

A subset of 150 fast5 files was obtained using the following:
```

> tar -xvf `ls | shuf -n1` # to obtain a random directory

Reviewer: Bit confused as to what this is doing?! Why do we need a random directory?

Reviewer: Maybe some more description?

cd <untarred_dir>
cp `ls | shuf -n 150` <output_dir> # to obtain random files
```

### Sample information

> 

Reviewer: ### Barcoded data in `fast5/barcoded/`

> | Barcoded Data (`test`)  |         	  |

Reviewer: ```suggestion
|                           |         	  |
```

|-------------------------|-------------|
| Flow Cell       	      | FLO-MIN106	|
| Kit	                    | SQK-DCS109	|
| Barcode Kit	            | EXP-NBD103	|

> | Non-Barcoded Data (`test_nobarcode`)|         	  |

Reviewer: ```suggestion
|                         |         	  |
```

> | Non-Barcoded Data (`test_nobarcode`)|         	  |

Reviewer: ```suggestion
### Non-barcoded data in `fast5/nonbarcoded/`
```
